app/assets/stylesheets/components/_contact-history.scss

```diff
@@ -1,5 +1,8 @@
-.contact-history {
+#contact-history {
   border-top: 2rem solid darken($color-grey-light, 50%);
   background-color: darken($color-grey-light, 10%);
   padding: 1rem;
 }
+#contact-history:focus {
```

Comment:
We attempted to add `:focus` styles. We were unable to trigger these when calling `.focus()` on the DOM element, but we suspect that's something that is easily fixable.